hubmap/frontend/src/components/home/LogInStatus.js

```diff
@@ -174,11 +174,11 @@ export default function LoggedInStatus() {
                     keepMounted
                     open={Boolean(anchorEl)}
                     onClose={handleClose}>
-                    <MenuItem  > Globus email: {email}
+                    <MenuItem  id="loggedinemail"> Globus email: {email}
                         <AccountCircleSharp className={classes.rightIcon} />
                     </MenuItem>
-                    <MenuItem onClick={handleCloseLogout} >
-                        <a href={API_URL + 'logout/'} style={{ textDecoration: 'none' }}>
+                    <MenuItem onClick={handleCloseLogout} id="logout-menuitem">
+                        <a id="logout" href={API_URL + 'logout/'} style={{ textDecoration: 'none' }}>
```

Comment:
Let's not add all these IDs: What matters is whether "Logout from Globus" is on the screen: test for that.

Reply:
@mccalluc the material-ui menus are hidden, until explicitly opened. So <a> tags are not accessible, as they are hidden. But then we also want to click logout and test it as per your issue #98. So we need ids. We are not adding explicitly, some had ids, some did not. so not sure if you noticed, menus are actually trigged by mousemove and opened and clicked. did you get a chance to run these tests ??

Reply:
As per our recent discussion: it seems question changes to "do we want to test elements that change/loaded on DOM events/user actions ?"